src/components/dialogs/network-modifications/vsc/hvdc-line-pane/vsc-hvdc-line-pane.tsx

```diff
@@ -220,6 +221,8 @@ const VscHvdcLinePane: FunctionComponent<VscHvdcLinePaneProps> = ({
                 {gridItem(droopField, 6)}
                 {gridItem(p0Field, 6)}
             </Grid>
+
+            <PropertiesForm isModification={isEquipementModification} />
```

Comment:
networkElementType ?

Reply:
Not for HVDC, that's why I changed the code of PropertiesForm